### ISSUE TITLE
Add: ABC 5 Minneapolis MN (KSTP)

### ISSUE THREAD

**Issue by freearhey** [iptv-org/iptv#12221]
### Channel ID

KSTPDT1.us

### Notes

Original request: #5447

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Sphinxroot** [comment]
#EXTINF:-1,kstp https://kstp.com/live-video/
https://fuel-streaming-prod01.fuelmedia.io/v1/sem/bd20854f-4027-4403-b2f1-5fa5ba886b42.m3u8

bd20854f-4027-4403-b2f1-5fa5ba886b42  might change

**Comment by freearhey** [comment]
https://fuel-streaming-prod01.fuelmedia.io/v1/sem/bd20854f-4027-4403-b2f1-5fa5ba886b42.m3u8 (shows looped video)

<img width="1416" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/083ef4dc-b1fd-4c3b-8a89-17da561d31d8">

**Comment by Sphinxroot** [comment]
yes unstable  i need to restart it at every adds  but m3u8 unchanged except session id


#EXTVLCOPT:http-reconnect=true
#EXTVLCOPT:network-caching=1000
#EXTVLCOPT:http-origin=https://kstp.com
